protoc-gen-grpc-gateway/gengateway/template.go

```diff
@@ -54,6 +55,26 @@ func (b binding) QueryParamFilter() queryParamFilter {
 	return queryParamFilter{utilities.NewDoubleArray(seqs)}
 }
 
+// FieldMaskField returns the golang-style name of the variable for a FieldMask, if there is exactly one of that type in
+// the message. Otherwise, it returns an empty string.
+func (b binding) FieldMaskField() string {
+	var fieldMaskField *descriptor.Field
+	for _, f := range b.Method.RequestType.Fields {
+		if f.GetTypeName() == ".google.protobuf.FieldMask" {
+			// if there is more than 1 FieldMask for this request, then return none
```

Comment:
I refrained from anything like that because I didn't want to assume that it's incorrect to have more than 1 fieldmask in a request proto. Also because I'm relatively new to Go templating and couldn't think of a more elegant solution :)

You could argue an error should be returned if none are found, as well, correct?